src/assets/custom/css/_sass/_training-hero.scss

```diff
@@ -0,0 +1,149 @@
+.training-hero__background-image {
+  background-size: cover;
+  opacity: 0.9;
+  position: relative;
+  width: 100%;
+
+  @include small {
+    height: 216.27px;
+  }
+
+  @include medium-and-large {
+    height: 433px;
+  }
+
+  @include extra-large {
+    height: 519.5px;
+  }
+}
+
+$background-block-height: ($vertical-overlap *  4);
+.training-hero__lower-background-block {
+  background-color: $slate;
+  position: relative;
+
+  @include small {
+    height: 2px + $background-block-height;
+  }
+
+  @include medium {
+    height: 6px + $background-block-height;
+  }
+
+  @include large {
+    height: 16px + $background-block-height;
+  }
+
+  @include extra-large {
+    height: 28px + $background-block-height;
+  }
+}
+
+.training-hero__content-container {
+  max-width: 50%;
+  position: relative;
+  color: white;
+  transform: translateY(-650px);
+
+  @include small {
+    padding-top: 85px;
+  }
+  
+  @include small-and-medium {
+    @include max-width--out-there;
+    margin-left: 24px;
+  }
+
+  @include medium {
+    padding-top: 40px;;
+  }
+
+  @include large-and-extra-large {
+    @include max-width--out-there;
+    padding-top: 30px;
+  }
+}
+
+.training-hero__title {
+  @include small {
+    @include dudler-bold;
+    display: flex;
+    justify-content: center;
+    text-align: center;
+  }
+
+  @include medium-large-and-extra-large  {
+    @include habib-bold;
+    text-align: left;
+  }
+}
+
+.training-hero__duration {
+  @include base-bold;
+  color: white;
+  text-transform: uppercase;
+
+  @include small {
+    display: flex;
+    justify-content: center;
+    text-align: center;
+  }
+}
+
+.training-hero__course-instructor-container{
+  display: flex;
+  padding: 50px 0;
+}
+
+.training-hero__course-instructor-avatar {
+  background-position: center;
+  background-size: cover;
+  height: $vertical-overlap + 10px;
+  width: $vertical-overlap;
+}
+
+.training-hero__course-instructor-info {
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-end;
+  padding-left: 20px;
+  transform: translateY(8px);
+}
+
+.training-hero__course-instructor-title {
+  @include ponti-bold;
+  color: white;
+}
+
+.training-hero__course-instructor {
+  @include mercer-bold;
+  color: white;
+}
+
+.training-hero__intro-container {
+  @include small {
+    @include base;
+
+    & .cta--primary-coral {
+      margin-right: 27px;
```

Comment:
We shouldn't need this. The gap should come from padding within it's container. The reason you need this margin on the right is because `training-hero__content-container` has a margin left pushing the content away from the left.